apps/ledger-live-desktop/src/renderer/screens/platform/v2/Catalog/Card/types.ts

```diff
@@ -1,6 +1,11 @@
 import { RecentlyUsedManifest } from "@ledgerhq/live-common/wallet-api/react";
+import { LiveAppManifest } from "@ledgerhq/live-common/platform/types";
 
-export interface PropsRaw {
+export interface PropsRawFullCard {
+  manifest: LiveAppManifest;
+  onClick: (manifest: RecentlyUsedManifest) => void;
+}
+export interface PropsRawMinimumCard {
   manifest: RecentlyUsedManifest;
```

Comment:
Why not simply do a `manifest: RecentlyUsedManifest | LiveAppManifest;` then you don't have to change everything